src/Home.jsx

```diff
@@ -61,11 +61,9 @@ const Home = () => {
         <ScriptNav setCurrentScript={setCurrentScript} currentScript={currentScript} scripts={scripts} setScripts={setScripts} />
         <Upload setCurrentScript={setCurrentScript} setScripts={setScripts} scripts={scripts} />
         <AddActor actors={actors} setActors={setActors}/>
-      </div>
-      <div className="row">
         <ActorList actors={actors} setActors={setActors}/>
-        <CharacterList currentScript={currentScript} />
       </div>
+        <CharacterAssignment actors={actors} currentScript={currentScript}/>
```

Comment:
There was an animation from the drag-and-drop library that was slowing down the rerender, I reduced it to almost zero, and that eliminated the delay. The docs say it can't be zero since there must still be an end drag event to trigger other updates. https://github.com/atlassian/react-beautiful-dnd/blob/master/docs/guides/drop-animation.md

https://user-images.githubusercontent.com/46881905/236970074-385fc318-9120-4238-9aad-07ac4906c363.mp4